app/scripts/ui.js

```diff
@@ -78,7 +78,7 @@ async function start() {
      * In case the UI is already rendered, only update the streams.
      */
     const messageListener = async (message) => {
-      if (message?.name === 'CONNECTION_READY') {
```

Comment:
Is the old `CONNECTION_READY` message still used for anything? Maybe it can be removed if not.

Reply:
Hi

Reply:
I removed it.